File: reference/Models/Models/PartidosJornadas.cs
using System.Runtime.Serialization;
using System;
using System.Text.Json.Serialization;
using Org.BouncyCastle.Asn1.Cms;

namespace dosxl.Models.Models
{
    [Serializable]
    [DataContract]
    public class PartidosJornadas
    {
        [DataMember(Name = "ID")]
        public virtual int id { set; get; }

        [JsonPropertyName("jornada")]
        public virtual Jornadas idJornada { set; get; }

        [JsonPropertyName("equipo1")]
        public virtual Equipo? idEquipo1 { set; get; }

        [JsonPropertyName("equipo2")]
        public virtual Equipo? idEquipo2 { set; get; }

        [DataMember(Name = "fechaHoraPartido")]
        public virtual DateTime fechaHoraPartido { set; get; }

        [DataMember(Name = "lugar")]
        public virtual string lugar { set; get; }

        [DataMember(Name = "marcadorEquipo1")]
        public virtual int marcadorEquipo1 { set; get; }

        [DataMember(Name = "marcadorEquipo2")]
        public virtual int marcadorEquipo2 { set; get; }

        [DataMember(Name = "estadoPartido")]
        public virtual string estadoPartido { set; get; }

        [DataMember(Name = "hora")]
        public virtual string hora { set; get; }

        [DataMember(Name = "fechaHoraInicioPartido")]
        public virtual DateTime fechaHoraInicioPartido { set; get; }

        [DataMember(Name = "fechaHoraFinPrimerTiempo")]
        public virtual DateTime fechaHoraFinPrimerTiempo { set; get; }

        [DataMember(Name = "fechaHoraInicioSegundoTiempo")]
        public virtual DateTime fechaHoraInicioSegundoTiempo { set; get; }

        [DataMember(Name = "fechaHoraFinPartido")]
        public virtual DateTime fechaHoraFinPartido { set; get; }

        [DataMember(Name = "fechaHoraPausa")]
        public virtual DateTime fechaHoraPausa { set; get; }

        [DataMember(Name = "pausaAcumuladaPrimertiempo")]
        public virtual double pausaAcumuladaPrimerTiempo { set; get; }

        [DataMember(Name = "pausaAcumuladaSegundoTiempo")]
        public virtual double pausaAcumuladaSegundoTiempo { set; get; }

        [DataMember(Name = "tipoPartido")]
        public virtual string tipoPartido { set; get; }

        [DataMember(Name = "idaOvuelta")]
        public virtual string idaOvuelta { set; get; }

    }
}

File: app/models/partidos_jornadas.py
from sqlalchemy import Boolean, Column, DateTime, ForeignKey, Integer, String
from sqlalchemy.orm import relationship

from app.database import Base


class PartidosJornadas(Base):
    """
    Modelo de partido programado entre dos equipos.
    Tabla: partidosjornadas
    """

    __tablename__ = "partidosjornadas"

    id = Column(Integer, primary_key=True, index=True)

    idJornada = Column(Integer, ForeignKey("jornadas.id"), nullable=True, index=True)

    idEquipo1 = Column(Integer, ForeignKey("equipo.id"), nullable=True, index=True)
    idEquipo2 = Column(Integer, ForeignKey("equipo.id"), nullable=True, index=True)
    grupoID = Column(Integer, nullable=True)

    fechaHoraPartido = Column(DateTime, nullable=True)
    hora = Column(String(100), nullable=True)
    lugar = Column(Integer, nullable=True)
    etiquetaCancha = Column(String(255), nullable=True)

    marcadorEquipo1 = Column(Integer, nullable=True)
    marcadorEquipo2 = Column(Integer, nullable=True)
    estadoPartido = Column(String(100), nullable=True)

    tipoPartido = Column(String(100), nullable=True)
    partidoSugerido = Column(Boolean, nullable=True)
    modificadoManual = Column(Boolean, nullable=True)
    currentStatePadel = Column(String(100), nullable=True)

    jornada = relationship("Jornadas", back_populates="partidos_jornada")
    equipo1 = relationship("Equipo", foreign_keys=[idEquipo1], back_populates="partidos_local")
    equipo2 = relationship("Equipo", foreign_keys=[idEquipo2], back_populates="partidos_visitante")
